crb.c

```diff
@@ -303,3 +279,32 @@ size_t crb_recv(struct tpmbuff *buf)
 	/* noop, currently send waits until execution is complete*/
 	return 0;
 }
+
+u8 crb_init(struct tpm *t)
+{
+	u8 i;
+	struct tpm_crb_intf_id_ext id;
+
+	for (i = 0; i <= TPM_MAX_LOCALITY; i++)
```

Comment:
I had to remove this from tis_init(). Trying to close some localities on some systems causes crashes. See:
a0ba8b6b999493e85b6d098dadadc3d5a419e060

Reply:
ack